src/Workleap.DomainEventPropagation.Subscription.PullDelivery/EventPullerService.cs

```diff
@@ -41,6 +41,9 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
 
     private class EventGridSubscriptionEventPuller
     {
+        // At the moment the Release api only supports these delays
+        private static readonly List<TimeSpan> SupportedReleaseDelays = [TimeSpan.FromSeconds(0), TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(60), TimeSpan.FromSeconds(600), TimeSpan.FromSeconds(3600)];
```

Comment:
Did you find those in some MS doc? If yes, can you add the link to it in comment plz

Reply:
I'll add the link, even though it's quite lengthy 😓

In case the content changes over time, here's a screenshot
![image](https://github.com/user-attachments/assets/148a58f1-e9a0-42fc-8c5b-ce369176c0d4)